aslprep/workflows/asl/fit.py

```diff
@@ -672,6 +672,10 @@ def init_asl_fit_wf(
             sloppy=config.execution.sloppy,
         )
 
+        if config.execution.sloppy and config.workflow.run_reconall:
+            coreg_node = asl_reg_wf.get_node('bbreg_wf.bbregister')
+            coreg_node.inputs.args = ['--nmax 5', '--subsamp 4']
```

Comment:
Looks like it needs to be a string. And I did check, you can do:

```suggestion
            asl_reg_wf.inputs.bbreg_wf.bbregister.args = "--nmax 5 --subsamp 4"
```